### PR TITLE
Use system time for start time in netplay.

### PR DESCRIPTION
**Pull request by RachelBryk**

### CODE REVIEW

**Comment by RachelBryk** [comment]
Alternative to #1861

File: Source/Core/Core/NetPlayServer.cpp [hunk 1]
```diff
@@ -648,7 +648,7 @@ bool NetPlayServer::StartGame()
 	// no change, just update with clients
 	AdjustPadBufferSize(m_target_buffer_size);
 
-	g_netplay_initial_gctime = CEXIIPL::GetGCTime();
+	g_netplay_initial_gctime = (u32)Common::Timer::GetLocalTimeSinceJan1970();
```
Comment (phire) [inline Source/Core/Core/NetPlayServer.cpp]: Perhaps it's not a good idea to cast this to a u32.

Might cause issues after 2106.

Comment (RachelBryk) [inline Source/Core/Core/NetPlayServer.cpp]: Doesn't matter, gc doesn't support that far anyway.

Comment (phire) [inline Source/Core/Core/NetPlayServer.cpp]: You sure? Based on the CEXIIPL::GetGCTime function it looks like the gamecube is good upto 2136 to me.

**Comment by RachelBryk** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**